src/xdocs/cmdline.xml.vm

```diff
@@ -96,26 +96,27 @@ java -D&lt;property&gt;=&lt;value&gt;  \
           <code>-s</code> option. Default value is 8.
         </li>
         <li>
-          <code>-t, --tree</code> - Prints Abstract Syntax Tree(AST) of the checked file. The option
-          cannot be used other options and requires exactly one file to run on to be specified.
+          <code>-t, --tree</code> - This option is used to display the Abstract Syntax Tree (AST)
+          <b>without any comments</b> of the specified file. It can only be used on a single file
+          and cannot be combined with other options.
         </li>
         <li>
-          <code>-T, --treeWithComments</code> - Prints Abstract Syntax Tree(AST) with comment nodes
-          of the checked file. The option cannot be used with other options and requires exactly
-          one file to run on to be specified.
+          <code>-T, --treeWithComments</code> - This option is used to display the Abstract Syntax
+          Tree (AST) <b>with comment nodes excluding Javadoc</b> of the specified file. It can only
+          be used on a single file and cannot be combined with other options.
         </li>
         <li>
-          <code>-J, --treeWithJavadoc</code> - Prints Abstract Syntax Tree(AST) with Javadoc nodes
-          and comment nodes of the checked file. Attention that line number and columns will not be
-          the same as it is a file due to the fact that each javadoc comment is parsed separately
-          from java file. The option cannot be used with other options and requires exactly one
-          file to run on to be specified.
+          <code>-J, --treeWithJavadoc</code> - This option is used to display the Abstract Syntax
+          Tree (AST) <b>with Javadoc nodes</b> of the specified file. It can only be used on a
+          single file and cannot be combined with other options. Note that the column numbers
+          between the 2 will not be in sync with the file's numbering due to the fact that each
```

Comment:
Is this related to https://github.com/checkstyle/checkstyle/issues/14780 ? If so remove it since it was approved as a bug.

Reply:
Removed.. but I was considering removing it as part of the fix for #14780 itself

Reply:
I wouldn't legitimatize this as being written in official documentation, especially if it is a bug. It is sometimes ok to add some things are not unsupported till another issue, but you'll be here a while if you have to write out everything that is a bug in open tickets.

Reply:
Got it :)

Now that this is merged, let's try #14759